/******************************************************************************
 * Copyright 2017 The Apollo Authors. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 *****************************************************************************/

#ifndef INCLUDE_CYBERTRON_COMMON_PROTO_INST_H
#define INCLUDE_CYBERTRON_COMMON_PROTO_INST_H

#include <memory>
#include "cybertron/proto/channel_filter.pb.h"
#include "cybertron/proto/channel_service.pb.h"
#include "cybertron/proto/chatter.pb.h"
#include "cybertron/proto/common_config.pb.h"
#include "cybertron/proto/common_geometry.pb.h"
#include "cybertron/proto/config_schema.pb.h"
#include "cybertron/proto/cyber_header.pb.h"
#include "cybertron/proto/dag_config.pb.h"
#include "cybertron/proto/errcode.pb.h"
#include "cybertron/proto/header.pb.h"
#include "cybertron/proto/meta_message.pb.h"
#include "cybertron/proto/meta_stat.pb.h"
#include "cybertron/proto/metric_alarm.pb.h"
#include "cybertron/proto/metric_service.pb.h"
#include "cybertron/proto/module_service.pb.h"
#include "cybertron/proto/parameter.pb.h"
#include "cybertron/proto/proto_desc.pb.h"
#include "cybertron/proto/record.pb.h"
#include "cybertron/proto/rule_desc.pb.h"
#include "cybertron/proto/sensor_imu.pb.h"
#include "cybertron/proto/sim_time.pb.h"
#include "cybertron/proto/simulator.pb.h"
#include "cybertron/proto/topology_changes.pb.h"
#include "cybertron/proto/whisper.pb.h"



#endif

